src/tdd/TransactionCalculator.java

```diff
@@ -0,0 +1,53 @@
+package tdd;
```

Comment:
Look at proposed changes in attachment, tests could have to be adjusted to account for corrected totals.
[TransactionCalculator.txt](https://github.com/zachj1582/tdd/files/4963232/TransactionCalculator.txt)